<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="15.2610" tests="3" failures="3">
  <testsuite name="Root Suite" timestamp="2021-11-15T12:18:48" tests="0" file="cypress/integration/pecode-test.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Pecode Automation test task" timestamp="2021-11-15T12:18:48" tests="3" time="15.2440" failures="3">
    <testcase name="Pecode Automation test task Check the login tab" time="0.0000" classname="Check the login tab">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `form`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `form`, but never found it.
    at loginForm.auth (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:136:8)
    at Context.eval (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:107:24)]]></failure>
    </testcase>
    <testcase name="Pecode Automation test task Verify that all the elements are present on page" time="0.0000" classname="Verify that all the elements are present on page">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `[class=&quot;wrapper&quot;]`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `[class="wrapper"]`, but never found it.
    at loginForm.elements (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:145:33)
    at Context.eval (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:110:24)]]></failure>
    </testcase>
    <testcase name="Pecode Automation test task Make sure a screenshot is taken when the test fails" time="0.0000" classname="Make sure a screenshot is taken when the test fails">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `[class=&quot;wrapper&quot;]`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `[class="wrapper"]`, but never found it.
    at loginForm.testFailed (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:154:33)
    at Context.eval (http://localhost:64328/__cypress/tests?p=cypress\integration\pecode-test.spec.js:113:24)]]></failure>
    </testcase>
  </testsuite>
</testsuites>